subprojects/wrapper/src/main/java/org/gradle/wrapper/Install.java

```diff
@@ -71,18 +89,28 @@ public File call() throws Exception {
 
                 verifyDownloadChecksum(configuration.getDistribution().toString(), localZipFile, distributionSha256Sum);
 
-                logger.log("Unzipping " + localZipFile.getAbsolutePath() + " to " + distDir.getAbsolutePath());
                 unzip(localZipFile, distDir);
 
                 File root = getAndVerifyDistributionRoot(distDir, safeDistributionUrl.toString());
                 setExecutablePermissions(root);
                 markerFile.createNewFile();
 
+                printReleaseSummary();
+
                 return root;
             }
         });
     }
 
+    private void printReleaseSummary() {
+        if (gradleVersion != null) { // appVersion could be null in custom distributions
+            logger.log("");
+            logger.log("Welcome to Gradle " + gradleVersion + "!");
+            logger.log("See what's new at https://gradle.org/releases#" + gradleVersion);
```

Comment:
:x: I believe that the message doesn't print the correct Gradle version. For example I built a local distribution, then used the distribution to create new Wrapper files for version 4.4. However, when running a task with the generated Wrapper files the version of my current distribution is used. This should be 4.4 and not 4.7-20180313213750+0000 right?

```
$ ./gradlew error
Downloading https://services.gradle.org/distributions/gradle-4.4-bin.zip
........................................................................

Welcome to Gradle 4.7-20180313213750+0000!
See what's new at https://gradle.org/releases#4.7-20180313213750+0000
```